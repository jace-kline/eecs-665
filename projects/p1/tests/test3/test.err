FATAL [1,52]: Illegal character \
FATAL [1,75]: Illegal character \
