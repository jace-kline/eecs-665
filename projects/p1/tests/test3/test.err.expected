FATAL [1,1]: Bad escape sequence in char literal
FATAL [1,52]: Illegal character \
